Wikipedia/Code/WikidataDescriptionEditingController.swift

```diff
@@ -80,10 +81,7 @@ enum WikidataPublishingError: LocalizedError {
             completion(result.error)
 
             if let authorized = authorized, authorized, result.error == nil {
-                DispatchQueue.main.async {
-                    NotificationCenter.default.post(name: WikidataDescriptionEditingController.DidMakeAuthorizedWikidataDescriptionEditNotification, object: nil)
```

Comment:
It'd be nice to consolidate the stored state for notifications, right now it's across `UserDefaults` and its own Core Data stack. Could this still post `.didMakeAuthorizedWikidataDescriptionEditNotification` and when that happens, the notifications controller writes the deadline date to its own Core Data stack? That way, the state is only in one place. The check for whether or not to make the request for new notifications would be if the date keyValue exists and is after the current date.

Reply:
That makes sense, I was thinking it might make sense to store it in defaults if in the future we wanted to show something based on whether the user made an authorized edit